### ISSUE TITLE
[1.10.2]Feroxia spores won't grow

### ISSUE THREAD

**Issue by DodgerWoman26** [teagan75/Metamorphoses#32]
No matter what I do- I can't get them to grow and unique crops book says once planted it will have details on each stage but nothing will show up in book.

**Comment by teagan75** [comment]
Feroxia spores are very irritating to grow. So you planted one and the Unique Crops book has no details?

**Comment by DodgerWoman26** [comment]
Yes. Planted a few. Nothing in book. Tried growing them almost every where. Even nether lol. Tried making a new book and still nothing. They won't work in the growth machine in the capitol quest line either. Wasn't sure if it was an issue or if I was doing something wrong. Never used unique crops before.

**Comment by teagan75** [comment]
They should work in the growth machine because I tested that. Did you add dirt and water to the machine?
